Library/Homebrew/cmd/audit.rb

```diff
@@ -28,12 +28,8 @@ def audit
     problem_count = 0
 
     strict = ARGV.include? "--strict"
-    if strict && ARGV.resolved_formulae.any? && RUBY_VERSION.split(".").first.to_i >= 2
-      require "cmd/style"
-      ohai "brew style #{ARGV.resolved_formulae.join " "}"
-      style
-    end
-
+    do_style = strict && ARGV.resolved_formulae.any? && RUBY_VERSION.split(".").first.to_i >= 2
```

Comment:
Nope... the `check_style` calls are moved way down below at the end of each individual formula audit. Running it here would either make the output separate, or give the results for many formulae, requiring additional parsing work.